.travis.sh

```diff
@@ -86,6 +86,8 @@ case "$JDK" in
   ;;
 9-ea | 9-ea-stable)
   # see https://bugs.openjdk.java.net/browse/JDK-8131041 about "java.locale.providers"
```

Comment:
@marchof ouch, and what you wrote isn't gonna work as is, and looks like attempt to construct the whole command in variable.

Anyway command has been divided on two, where comments are above of respective parts - see updated commit. Seems quite acceptable to me given that actually all this should be gone eventually.